app/bundles/LeadBundle/Config/config.php

```diff
@@ -739,6 +754,13 @@
                     \Mautic\LeadBundle\Entity\Lead::class,
                 ],
             ],
+            'mautic.lead.repository.frequency_rule' => [
+                'class'     => Doctrine\ORM\EntityRepository::class,
```

Comment:
It is better to provide a specific class - just create a `FrequencyRuleRepository` even if it will be empty now.

Reply:
Will update

Reply:
Updated